### PR TITLE
Add january meetup report

### PR DESCRIPTION
**Pull request by ChillarAnand**

### CODE REVIEW

File: _posts/2016-01-30-salt-stack-workshop.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post.html
+title: January 2015 workshop report
+tags: [meetup, talks]
```
Comment (kracekumar) [inline _posts/2016-01-30-salt-stack-workshop.md]: Replace tag `talks` with  `workshop`.

File: _posts/2016-01-30-salt-stack-workshop.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+---
+
+We have kickstarted this year's meetup with  [salt stack workshop](http://www.meetup.com/BangPypers/events/222071834/) at [mavenhive technologies](http://www.mavenhive.in/).
```
Comment (kracekumar) [inline _posts/2016-01-30-salt-stack-workshop.md]: `We kickstarted` should be suffice.

Comment (kracekumar) [inline _posts/2016-01-30-salt-stack-workshop.md]: Head count was 23.

**Comment by ChillarAnand** [comment]
@kracekumar  Updated

**Pull request merged**